config/config.go

```diff
@@ -160,10 +202,17 @@ func Default() *Config {
 		},
 
 		Cache: CacheConfig{
+			Enabled: false,
+			Backend: CacheMemory,
+			TTL:     1 * time.Minute,
 			Memory: MemoryCacheConfig{
-				Enabled:          false,
-				Expiration:       -1,
-				EvictionInterval: 10 * time.Minute,
+				EvictionInterval: 5 * time.Minute,
```

Comment:
> TTL replaces expiration here and expiration

Yeah so `TTL = expiration from go-cache terminology`. Previously the default config used `-1` for expiration which meant items never expired automatically. Since now we are more heavily reliant on TTL since we're moving the caching logic up a layer, I figured it made sense to set a 'sane' 1m default instead across the board.

> evicition interval are a gocache paradigm

Yeah. `go-cache` lets you set how often the 'cleanup' routine runs to deleted expired items from the cache, unlike redis (see below) which seems to manage this process it self

> Redis cache accepts TTL and uses it to refresh the cache when there's a read + TTL expiration

Yeah so when we go to read from Redis and it sees that the TTL has expired it acts as a 'miss' and so we go read from the DB and then repopulate the cache item. Redis randomly also deletes expired items from the redis db to clear space, but it doesnt auto-repopulate the cache itself: https://redis.io/commands/expire/#how-redis-expires-keys